self-tests.yml

```diff
@@ -604,3 +607,14 @@ buildvariants:
       mongodb_url: https://downloads.mongodb.com/linux/mongodb-linux-arm64-enterprise-ubuntu1604-4.0.3.tgz
     tasks:
       - name: ".agent .test"
+  - name: linux-docker
+    display_name: ArchLinux (Docker)
+    run_on:
+      - archlinux-test
+    expansions:
+      goos: linux
+      goarch: amd64
+      gobin: /opt/golang/go1.9/bin/go
+      goroot: /opt/golang/go1.9
```

Comment:
Arch linux doesn't have mongodb provided toolchain docker, just use the go that is installed with packages.

Reply:
done.